src/main/java/org/kairosdb/core/http/rest/json/DataPointsParser.java

```diff
@@ -63,6 +66,9 @@ public int getIngestTime()
 	private int dataPointCount;
 	private int ingestTime;
 
+	@Inject
+	private Tracer tracer;
```

Comment:
Nitpick: if we're always using GlobalTracer.get(), we won't need to inject a specific Tracer.

Reply:
Nice catch. I thought i had already removed it.